docs/deployments/import-and-export-users.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Import and export users
+description: Learn how to import users to and export users from your Clerk application.
+---
+
+# Import and export users
+
+If you need some guidance or have 200k+ users, please reach out to [support](https://clerk.com/support).
+
+## Import users
+
+You can use the [Backend API](https://clerk.com/docs/reference/backend-api/tag/Users#operation/CreateUser) to create users and import them from your previous provider. 
+
+## Export users
```

Comment:
As the top line is mentioning importing 200K+ (which is someone migrating to Clerk), it might be worth adding a line about requesting an export of users for migrating away. IE "If you wish to receive a CSV with all user data including their hashed passwords, please reach out to support."